src/server/controllers/questions.js

```diff
@@ -69,6 +75,40 @@ const optionsRead = {
   ],
 };
 
+async function getVotes(req, res, next) {
+  if (req.get('Content-Type') === 'text/csv') {
+    const { resource } = req.locals;
+
+    try {
+      const question = await Question.findOne({
+        rejectOnEmpty: true,
+        where: { festivalId: resource.id },
+      });
+
+      const votes = await Vote.findAll({
+        rejectOnEmpty: true,
+        where: { festivalQuestionChainId: question.chainId },
```

Comment:
I think similar here, but that this should toggle for artwork questions, and if `type === 'artwork` where `{artworkQuestionChainId: question.chainId}`